includes/helpers.php

```diff
@@ -5,3 +5,20 @@ function ctype_alnum( $text ){
 		return ( preg_match('~^[0-9a-z]*$~iu', $text ) > 0 );
 	}
 }
+
+if ( ! function_exists( 'update_entry_meta' ) ) {
+
+	/**
+	 * Update Entry meta
+	 *
+	 * Allows a meta to be altered before posting to Eloqua.
+	 *
+	 * @param  string $key  The key of the metadata.
+	 * @param  string $lead The value of the metadata.
+	 * @param  object $form The form that the metadata belongs to.
+	 * @return string       Returns the value of the entry meta.
+	 */
+	function update_entry_meta( $key, $lead, $form ) {
```

Comment:
@solepixel looking further into the API documentation https://www.gravityhelp.com/documentation/article/gform_entry_meta/ if does seem you need to define your own custom function. 

Judging by the context of the call, I would say that the `update_entry_meta function that has been defined in the helper class should return `true`.

Let me know if you agree.